app/src/main/java/com/akshay/newsapp/koin/AppInjector.kt

```diff
@@ -0,0 +1,34 @@
+package com.akshay.newsapp.koin
```

Comment:
I would suggest to the use the same package `com.akshay.newsapp.di` for all the Koin DI stuff and like before `AppInjector` can live in following suggestion:
```suggestion
package com.akshay.newsapp.di.base
```

Reply:
Done. The di package is now same as before.